kong/db/strategies/postgres/init.lua

```diff
@@ -6,6 +6,8 @@ local utils         = require "kong.tools.utils"
 local new_tab       = require "table.new"
 local clear_tab     = require "table.clear"
 
+cjson.encode_number_precision(16)
+cjson_safe.encode_number_precision(16)
```

Comment:
Will this impact all of `cjson`'s instances globally since `cjson` here is not `new`'ed?

Reply:
Also, we should define the magic number inside `constants.lua` as something like `CJSON_MAX_PRECISION = 16` instead of hard coding.